### ISSUE TITLE
Build fails for the custom environment with conda

### ISSUE THREAD

**Issue by vishalmhjn** [MicrosoftLearning/mslearn-azure-ml#58]
# Module: Work with environments in Azure Machine Learning
#### Step: Cell 10 and 11

Description of issue:

Building the custom image with conda takes takes forever and eventually times out with following build_log:
```
Collecting package metadata (repodata.json): ...working... 
done
Solving environment: ...working... 
```
Updating the Python version from 3.7 to 3.11 in step 10 (conda-env.yml) helps.

```
%%writefile src/conda-env.yml
name: basic-env-cpu
channels:
  - conda-forge
dependencies:
  - python=3.11
  - scikit-learn
  - pandas
  - numpy
  - matplotlib
```

Another observation is that jobs on compute clusters are being shown as "preparing" for little more than 10 minutes, which is kind of slow. I am not sure if this is normal in Azure.

**Comment by suketunayak** [comment]
Yes Custom Environment creation is going on more than 45 minutes still running....

![image](https://github.com/MicrosoftLearning/mslearn-azure-ml/assets/12823966/9b400d65-0cc5-41ac-a1d1-b85322c589b5)

Job is also in Preparing Status since more than 45 minutes

![image](https://github.com/MicrosoftLearning/mslearn-azure-ml/assets/12823966/3adfc935-e1ce-4506-9fff-6fac91895548)
